src/components/Layout/Topbar/TopBar.jsx

```diff
@@ -0,0 +1,74 @@
+import { HStack, Spacer, Icon, Heading, Button, Flex, Text } from '@chakra-ui/react';
+import PropTypes from 'prop-types'; // ES6
+import { FaTeamspeak } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+
+function TopBar({ isLogged, setIsLogged }) {
+  let links;
+
+  const handleLogout = () => {
+    setIsLogged(false);
+  };
+
+  const makeLinkButton = (el, index) => {
```

Comment:
We should extract this into a separate component, no need to have a function for it :)